pymobiledevice3/exceptions.py

```diff
@@ -345,3 +345,8 @@ class NotEnoughDiskSpaceError(PyMobileDevice3Exception):
 class RSDRequiredError(PyMobileDevice3Exception):
     """ The requested action requires an RSD object """
     pass
+
+
+class SysdiagnoseTimeoutError(PyMobileDevice3Exception, TimeoutError):
+    """ Timeout collecting new sysdiagnose archive """
+    pass
```

Comment:
I'm on the fence about raising an error.
Also unsure about the best error name if raising an error is acceptable.